fact_extractor/plugins/unpacking/draytek_vigor_167/code/draytek_vigor_167.py

```diff
@@ -0,0 +1,35 @@
+'''
+This plugin extracts the squashfs root filesystem in Draytek Vigor 167 firmware containers.
+'''
+import struct
+from pathlib import Path
+from common_helper_files import write_binary_to_file
+
+NAME = 'Draytek Vigor 167'
+MIME_PATTERNS = ['firmware/draytek-vigor-167']
+VERSION = '0.1'
+
+header_layout = '> 4s I I I 64x I I 49x I I 117x'
+header = struct.Struct(header_layout)
```

Comment:
constants should be `UPPERCASE`

```suggestion
HEADER_LAYOUT = '> 4s I I I 64x I I 49x I I 117x'
HEADER = struct.Struct(HEADER_LAYOUT)
```

Also, I'm not quite sure if we should use such a large unpacking string without further documentation in code. It is rather hard to understand what the single values express. It would be nice to understand how the header is structured at all.
Could you please visualize the layout in the comments using, e.g., [asciiflow](https://asciiflow.com/)?